pages/home.page.js

```diff
@@ -0,0 +1,8 @@
+const PAGE_TIMEOUT = 10000;
+
+class HomePage {
+       
+        get welcomeMsg () { return $('[class="active"]'); }
```

Comment:
This selector seems a bit flaky, as it makes reference to a rather general class that can potentially be used on any other element. Consider using a more robust selector (something that more closely refers to that specific element and that has more potential for unicity )